docs/UserGuide.md

```diff
@@ -237,6 +237,22 @@ _This command allows you to undo your last command, restoring the ConnectCare ap
 
 Format: `undo`
 
+**Walkthrough:**
+
+The screenshots below are what you would expect when using the `undo` command.
+
+In this example, after using the `add` command to add a new client `bertrand`, I wish to undo this addition as bertrand will not be assigned to me.
+&nbsp;
+
+<img src="images/user-guide/undo_command_before1.png" alt="Update command example" />
+&nbsp;
```

Comment:
A caption should be added below the image to describe what is happening (see delete for example).